src/kili/entrypoints/mutations/label/__init__.py

```diff
@@ -57,6 +58,9 @@ def create_predictions(
             model_name: The name of the model that generated the predictions
             asset_id_array: The internal IDs of the assets for which we want to add predictions.
             disable_tqdm: Disable tqdm progress bar.
+            overwrite: when uploading predictions or inference labels, if True,
+                it will overwrite existing predictions and inference labels
```

Comment:
so here we are in the create_predictions method, so we shouldn't even be talking about inference labels here